docs/HowTo/Interact/APIs/Authentication.md

```diff
@@ -3,20 +3,29 @@ description: Hyperledger Besu authentication and Authorization for JSON-RPC
 
 # Authentication and Authorization for JSON-RPC
 
-Authentication identifies a user based on a username and password. Authorization verifies whether the user has
-access to the JSON-RPC method they are requesting.  
+Authentication identifies a user, and authorization verifies user 
+access to requested JSON-RPC methods. Users are verified using a [JWT token](https://jwt.io/introduction/)
 
-Hyperledger Besu uses the username and password to authenticate users and 
-[JWT tokens](https://jwt.io/introduction/) to authorize JSON-RPC requests. 
+Hyperledger Besu supports two mutually exclusive authentication methods:
+
+* [Username and password](#username-and-password-authentication)
+* [JWT public key](#jwt-public-key-authentication).
+
+JWT tokens are created internally with username and password authentication, and externally with JWT public key authentication.
 
 !!! important 
     To prevent interception of authentication credentials and authenticated tokens, make authenticated requests over HTTPS. 
     We recommended production deployments are run behind a network layer that provides SSL termination. 
     Besu does not provide a HTTPS connection natively.
 
-## Credentials File 
 
-The credentials file is a `toml` file defining user details and the JSON-RPC methods to which they have access. 
+## Username and Password Authentication
+
+Enable authentication from the command line. Supply the credentials file and send a request to the `/login` endpoint using the username and password to obtain the JWT token.
```

Comment:
earlier we said these were mutually exclusive. Can we be clearer about how username/password relates to JWT?

Reply:
@jframe, could you perhaps help me describe how the username/password relates to JWT?

Reply:
I'm not sure what to add beyond what is already here but I'll try.
The username/password is only used to authenticated to the login endpoint in Besu. And the login endpoint will issue a JWT if the login is successful.
This is mutually exclusive with using the public key authentication approach as the the username/password are not used when using the public key. And the login endpoint is disabled.
You can only use the login endpoint with username/password or create your JWT tokens and supply a public key.

Reply:
Done.